src/test/java/net/snowflake/client/jdbc/ConnectionAlreadyClosedIT.java

```diff
@@ -42,7 +42,7 @@ public void testClosedConnection() throws Throwable {
     expectConnectionAlreadyClosedException(connection::getTransactionIsolation);
     expectConnectionAlreadyClosedException(connection::getWarnings);
     expectConnectionAlreadyClosedException(connection::clearWarnings);
-    expectConnectionAlreadyClosedException(() -> connection.nativeSQL("sekect 1"));
+    expectConnectionAlreadyClosedException(() -> connection.nativeSQL("select 1"));
```

Comment:
This is notable.

I understand that the code doesn't really run anyway, but it seems unlikely that you were trying to test w/ a syntax error.